src/views/TradingCompetition/components/ClaimModal/index.tsx

```diff
@@ -56,7 +56,7 @@ const ClaimModal: React.FC<CompetitionProps> = ({ onDismiss, onClaimSuccess, use
         </Text>
         <Flex mt="16px" alignItems="center">
           {/* achievements */}
-          <Image src={`/images/achievements/${achievement.image}`} width={25} height={25} />
+          <Image src={`/images/achievements/${achievement[0].image}`} width={25} height={25} />
```

Comment:
I remember this thrown some error previously, is it just to avoid error for now?
Otherwise I think there is either achievement or undefined returned, no?
Just need to not forget to change it later

Reply:
got it thanks!